### PR TITLE
fixrotations fixes rotations properly again

### PR DESCRIPTION
**Pull request by 20kdc**
## About the PR

For packedstation mapping mainly

Specific objects fixrotations will now mess with:
- Anything with a *round-start enabled* OccluderComponent (done this way so it'll ignore edge firelocks/etc.)
- Anything with a CableComponent (implicitly assuming this is paired with CableVis)
- Anything with a SharedCanBuildWindowOnTopComponent
- There's now a tag to force fixrotation on anything

no changelog notes, only relevant to mappers

### CODE REVIEW

**Review by ShadowCommander — APPROVED** [review]

**Comment by mirrorcult** [comment]
does this fix the issue of thindows with occluders getting rotated by fixrotations?

**Comment by mirrorcult** [comment]
or are rotated/non-fulltile occluders sufficiently broken that it's probably fine to ignore that

**Comment by 20kdc** [comment]
Oh, right, it doesn't - and they're still broken anyway, I need to get back to my occluders reform at some point
I'll put in a tag to exempt them

**Comment by 20kdc** [comment]
this should avoid directional tinted windows being broken

**Review by ShadowCommander — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -225,3 +225,10 @@
 
 - type: Tag
   id: NukeDisk
+
+- type: Tag
+  id: ForceFixRotations # fixrotations command WILL target this
+
+- type: Tag
+  id: ForceNoFixRotations # fixrotations command WON'T target this
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:234]: Alphabetise please, don't continue people's bad habits.

Comment (20kdc) [reply Resources/Prototypes/tags.yml]: Done

**Comment by 20kdc** [comment]
Alphabetized the tags.

**Pull request merged**